qutip/tests/core/test_superoper.py

```diff
@@ -67,6 +68,25 @@ def testOperatorVector(self):
         rho2 = qutip.vector_to_operator(qutip.operator_to_vector(rho1))
         np.testing.assert_allclose(rho1.full(), rho2.full(), 1e-8)
 
+    def testsuperrep(self):
+        N = 3
+        rho1 = qutip.rand_dm(N)
+        as_vec = qutip.operator_to_vector(rho1)
+        assert as_vec.superrep == 'super'
+
+        with pytest.raises(TypeError) as err:
+            as_vec.superrep = ""
+            qutip.vector_to_operator(as_vec)
+        assert err.value.args[0] == ("only defined for operator-kets "
+                                     "in super format")
+
+        with pytest.raises(TypeError) as err:
+            as_vec.superrep = ""
```

Comment:
It's a bit odd setting `superrep = ""` here because 1) it has already been set to `""` above and 2) its not directly relevant to the test. I would perhaps move `as_vec.superrep = ""` above both `with pytest.raises(...) ...` blocks and add a short comment on why it is being set to `""`.

Reply:
Fixed!